src/components/volume-viewer/VolumeViewer.jsx

```diff
@@ -0,0 +1,24 @@
+import { Container } from "react-bootstrap";
+import { Sidebar } from "primereact/sidebar";
+
+import VolumeRenderer from "./VolumeRenderer";
+import Controls from "../controls/Controls";
+
+export default function VolumeViewer(props) {
+  const { sidebarVisible, setSidebarVisible } = props;
+
+  return (
+    <Container fluid id="visualizer" className="mb-3">
+      <VolumeRenderer />
+
+      <Sidebar
```

Comment:
since the sidebar is on the left, can we put it above the `VolumeRenderer` so that the flow matches the natural layout closer (even though magic is making rendering happen on the left anyway)?

Reply:
I moved `Sidebar` into App, right above `VolumeRenderer`. I think it makes more sense in the `App.jsx` layout itself but let me know if I should leave it in `VolumeViewer

Reply:
With the current layout, `App` probably makes the most sense, but when we start carving out the viewer as a standalone component with a built-in sidebar, it will probably make sense to move it back under that umbrella.